### ISSUE TITLE
the app window is not loading when i run yarn dev

### ISSUE THREAD

**Issue by Optimus2297** [responsively-org/responsively-app#599]
# 🐞 bug report

### ✍️ Description

i have installed all the dependencies using yarn and when i am trying to open the app in dev mode using yarn dev the window of the app is not loading. Its not giving any errors also in the console.

### 🕵🏼‍♂️ Is this a regression?

It was working properly and suddenly what happened i dont know i didt change any code and then this problem appeared

### 🔬 Minimal Reproduction

yarn dev

### 🌍 Your Environment

windows

### 🔥 Exception or Error or Screenshot

<pre><code>
D:\responsively-app-master\desktop-app>yarn dev
yarn run v1.22.4
$ cross-env START_HOT=1 node -r @babel/register ./internals/scripts/CheckPortInUse.js && cross-env START_HOT=1 yarn start-renderer-dev
$ cross-env NODE_ENV=development webpack-dev-server --config configs/webpack.config.renderer.dev.babel.js
Starting Main Process...
$ cross-env HOT=1 NODE_ENV=development electron -r @babel/register ./app/main.dev.js

(node:10008) ExtensionLoadWarning: Warnings loading extension at C:\Users\bhaskarbhavsaheb.h\AppData\Roaming\Electron\extensions\fmkadmapgofadopljbjfkapdkoienihi: Unrecognized manifest key 'browser_action'. Unrecognized manifest key 'minimum_chrome_version'. Unrecognized manifest key 'update_url'. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
(node:10008) ExtensionLoadWarning: Warnings loading extension at C:\Users\bhaskarbhavsaheb.h\AppData\Roaming\Electron\extensions\lmhkpmbekcpmknklioeibfkpmmfibljd: Unrecognized manifest key 'commands'. Unrecognized manifest key 'homepage_url'. Unrecognized manifest key 'page_action'. Unrecognized manifest key 'short_name'. Unrecognized manifest key 'update_url'. Permission 'notifications' is unknown or URL pattern is malformed. Permission 'contextMenus' is unknown or URL pattern is malformed. Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
[Browsersync] Copy the following snippet into your website, just before the closing </body> tag
<script async id="__bs_script__" src="https://localhost:12719/browser-sync/browser-sync-client.js?v=2.26.10"></script>

10:49:21.638 > Skip checkForUpdatesAndNotify because application is not packed
</code></pre>

**Comment by jjavierdguezas** [comment]
I can't reproduce this

I started a new windows sandbox session, installed node, git, yarn and vscode, then cloned the master branch, ran `yarn` then `yarn dev` and the app started fine
Windows showed some dialog boxes about the firewall, and I just clicked OK (I think those appear only the first time)

Is there something different or unique about your environment?
Did you try `rm -rf node_modules` and start over?

**Comment by Optimus2297** [comment]
> I can't reproduce this
> 
> I started a new windows sandbox session, installed node, git, yarn and vscode, then cloned the master branch, ran `yarn` then `yarn dev` and the app started fine
> Windows showed some dialog boxes about the firewall, and I just clicked OK (I think those appear only the first time)
> 
> Is there something different or unique about your environment?
> Did you try `rm -rf node_modules` and start over?

yes i did   Did you try `rm -rf node_modules` and start over? this but still facing the issue, and if i do "yarn dev .", the the window opens but evrything is black nothing loads except devtools

**Comment by manojVivek** [comment]
Do you see any errors in the dev tools console?

**Comment by Optimus2297** [comment]
> Do you see any errors in the dev tools console?

previously i was not having any errors as you see in the code i have pasted in the issue, but now i am getting these things also
"[13048:0602/121106.823:ERROR:CONSOLE(115)] "File system error: A requested file or directory could not be found at the time an operation was processed. when requesting entry ''", source: devtools://devtools/bundled/persistence/persistence.js (115)
[13048:0602/121106.823:ERROR:CONSOLE(115)] "File system error: A requested file or directory could not be found at the time an op"

**Comment by GoonerBrian** [comment]
Was a solution ever found on this issue? I'm having the same issue. I'm not on Windows. I have an M1 Mac. Could that be the issue?
